### ISSUE TITLE
Android Auto keeps crashing

### ISSUE THREAD

**Issue by GittyWarlock** [ProtonAOSP/android_manifest#7]
Whenever I try and launch Android Auto, it apparently keeps crashing with the update. it also refuses to connect via USB. Completely unusable on android 12.

**Comment by kdrag0n** [comment]
Please get a logcat right after it crashes.

**Comment by GittyWarlock** [comment]
One order of hotfix, please. Scattered, Smothered, and Covered.
[dump.txt](https://github.com/ProtonAOSP/android_manifest/files/7500964/dump.txt)

**Comment by kdrag0n** [comment]
Try granting the "Nearby devices" permission to Android Auto.

**Comment by GittyWarlock** [comment]
Really?! That simple?! Yeah, It's working now. I've also been given a rude awakening by the phone app in Android Auto. Apparently, the phone permission needs to be allowed for it to function. After reviewing the other permissions, I've decided to grant all permissions to Android Auto. I've got a bad feeling that because it's one of those few apps that does it all, any permissions that are withheld will tend to break functionality. Feel free to post this fix in the FAQ's. Just make sure that people disable Google Analytics in the Android Auto app before granting the app everything short of root permissions. 

Thanks for your patience, and for being a pleasant developer to work with Danny. Just don't let it go to your head. The community needs people like you to remain humble and considerate.
